spec/observables/of-spec.ts

```diff
@@ -3,19 +3,21 @@ import * as Rx from '../../dist/cjs/Rx';
 import {ArrayObservable} from '../../dist/cjs/observable/ArrayObservable';
 import {ScalarObservable} from '../../dist/cjs/observable/ScalarObservable';
 import {EmptyObservable} from '../../dist/cjs/observable/EmptyObservable';
+import { Observable } from '../../dist/cjs/Observable';
+import '../../dist/cjs/add/observable/of';
+import '../../dist/cjs/add/operator/concatAll';
 import marbleTestingSignature = require('../helpers/marble-testing'); // tslint:disable-line:no-require-imports
 
 declare const { asDiagram };
 declare const expectObservable: typeof marbleTestingSignature.expectObservable;
 declare const rxTestScheduler: Rx.TestScheduler;
-const Observable = Rx.Observable;
 
 /** @test {of} */
 describe('Observable.of', () => {
   asDiagram('of(1, 2, 3)')('should create a cold observable that emits 1, 2, 3', () => {
-    const e1 = Observable.of(1, 2, 3)
+    const e1 = Rx.Observable.of(1, 2, 3)
```

Comment:
This seems wrong. We're importing `Observable` above, we should probably use that, no?